Gopkg.lock

```diff
@@ -614,99 +479,33 @@
   ]
-  pruneopts = "UT"
   revision = "552e98edab5d620205ff1a8960bf52a5a10aad03"
   version = "v2.1.5"
 
 [[projects]]
-  digest = "1:cacb98d52c60c337c2ce95a7af83ba0313a93ce5e73fa9e99a96aff70776b9d3"
   name = "gopkg.in/yaml.v2"
   packages = ["."]
-  pruneopts = "UT"
   revision = "a5b47d31c556af34a302ce5d659e6fea44d90de0"
 
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  input-imports = [
-    "github.com/dgrijalva/jwt-go",
-    "github.com/fabric8-services/fabric8-cluster-client/cluster",
-    "github.com/fabric8-services/fabric8-cluster/design",
-    "github.com/fabric8-services/fabric8-common/gocksupport",
-    "github.com/fabric8-services/fabric8-common/httpsupport",
-    "github.com/fabric8-services/fabric8-common/test/suite",
-    "github.com/fabric8-services/fabric8-notification/design",
-    "github.com/fabric8-services/fabric8-tenant/design",
-    "github.com/fabric8-services/fabric8-wit/design",
-    "github.com/fzipp/gocyclo",
-    "github.com/getsentry/raven-go",
-    "github.com/goadesign/goa",
-    "github.com/goadesign/goa/client",
-    "github.com/goadesign/goa/cors",
-    "github.com/goadesign/goa/design",
-    "github.com/goadesign/goa/design/apidsl",
-    "github.com/goadesign/goa/encoding/form",
-    "github.com/goadesign/goa/goagen",
-    "github.com/goadesign/goa/goagen/codegen",
-    "github.com/goadesign/goa/goagen/gen_app",
-    "github.com/goadesign/goa/goagen/gen_client",
-    "github.com/goadesign/goa/goagen/gen_controller",
-    "github.com/goadesign/goa/goagen/gen_js",
-    "github.com/goadesign/goa/goagen/gen_swagger",
-    "github.com/goadesign/goa/goagen/utils",
-    "github.com/goadesign/goa/goatest",
-    "github.com/goadesign/goa/logging/logrus",
-    "github.com/goadesign/goa/middleware",
-    "github.com/goadesign/goa/middleware/gzip",
-    "github.com/goadesign/goa/middleware/security/jwt",
-    "github.com/goadesign/goa/uuid",
-    "github.com/gojuno/minimock",
-    "github.com/gojuno/minimock/cmd/minimock",
-    "github.com/golang/lint/golint",
-    "github.com/jinzhu/gorm",
-    "github.com/jstemmer/go-junit-report",
-    "github.com/jteeuwen/go-bindata/go-bindata",
-    "github.com/lib/pq",
-    "github.com/pilu/fresh",
-    "github.com/pkg/errors",
-    "github.com/prometheus/client_golang/prometheus",
-    "github.com/satori/go.uuid",
-    "github.com/sergi/go-diff/diffmatchpatch",
-    "github.com/sirupsen/logrus",
-    "github.com/spf13/cobra",
-    "github.com/spf13/viper",
-    "github.com/stretchr/testify/assert",
-    "github.com/stretchr/testify/require",
-    "github.com/stretchr/testify/suite",
-    "github.com/wadey/gocovmerge",
-    "golang.org/x/crypto/bcrypt",
-    "golang.org/x/net/context",
-    "golang.org/x/net/websocket",
-    "golang.org/x/oauth2",
-    "golang.org/x/oauth2/github",
-    "gopkg.in/h2non/gock.v1",
-    "gopkg.in/square/go-jose.v2",
-    "gopkg.in/yaml.v2",
-  ]
+  inputs-digest = "91c2d17f41b78bab590d1ae93d7a2e41171135e7783b7cc6f8ccf914167a17ec"
```

Comment:
Not sure why your dep keep changing this file.. Looks like you randomly use different dep versions ;) But it's not that critical.